### ISSUE TITLE
how to run dev mode for watching the page immediately?

### ISSUE THREAD

**Issue by Taosome** [chainapsis/keplr-wallet#633]
**Describe the bug**
I want to run keplr wallet extension locally on my computer. After following readme.md, i run yarn build successfully. But if i run 'yarn dev', i'm stuck！

**To Reproduce**
1. yarn bootstrap
2. yarn build
3. yarn dev

**Expected behavior**
Webpack can run a local server such as 127.0.0.1:8080 , then i could see the keplr wallet page through chrome.

**Device details (please complete the following information):**
 - OS: WSL: Ubuntu-20.04
 - Keplr Version: 0.11.37

**Additional context**
Is there anyone can help me pls?

**Comment by HeesungB** [comment]
Do you install protoc?

This repo requires protoc to be installed. Check [Install protobuf](https://grpc.io/docs/protoc-installation/) for details.

**Comment by Taosome** [comment]
@HeesungB Yeah, I've already install protoc. Whatever i can even run 'yarn build' successfully.

**Comment by HeesungB** [comment]
@Taosome 
Could you execute this command? 
`yarn workspace @keplr-wallet/extension dev`
